tests/visualization_stress/solution/solution-00000.0000.gnuplot

```diff
@@ -1,455 +1,455 @@
 # This file was generated by the deal.II library.
-
-
+# Date =  2020/8/5
+# Time =  21:36:16
 #
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
-# <x> <y> <velocity> <velocity> <p> <T> <stress_xx> <stress_yy> <stress_xy> 
-0 0 0 0 9.99962 1 9.99962 9.99962 42 
-0.125 0 0 0 10 0 10 9.99997 41.9999 
+# <x> <y> <velocity> <velocity> <p> <T> <stress_xx> <stress_xy> <stress_yx> <stress_yy> 
+0 0 0 0 9.99962 1 9.99962 42 42 9.99962 
+0.125 0 0 0 10 0 10 41.9999 41.9999 9.99997 
```

Comment:
The last 3 components of the old file were the xx, yy, and xy components. The last four components of the new file are the xx, xy, yx, yy components of the tensor.